packages/typescript/ai/tests/middlewares/fake-otel.ts

```
@@ -0,0 +1,237 @@
+import type {
+  Attributes,
+  AttributeValue,
```

ESLint

[error] Member 'AttributeValue' of the import declaration should be sorted alphabetically.

(sort-imports)